designsafe/apps/accounts/views.py

```diff
@@ -467,8 +467,12 @@ def email_confirmation(request, code=None):
                 if tas.verify_user(user['id'], code, password=password):
                     logger.info('TAS Account activation succeeded.')
                     from django.conf import settings
-                    check_or_create_agave_home_dir.apply_async(args=(user.username, settings.AGAVE_STORAGE_SYSTEM))
-                    check_or_create_agave_home_dir.apply_async(args=(user.username, settings.AGAVE_WORKING_SYSTEM))
+                    systems_to_configure = [
+                        {"system_id": settings.AGAVE_STORAGE_SYSTEM, "create_credentials": True},
+                        {"system_id": settings.AGAVE_WORKING_SYSTEM, "create_credentials": False},
```

Comment:
Should we include the base `cloud.data` system for credential creation too? In the case that we ever want to use that system to list community or public data for example?

Reply:
@jarosenb @rstijerina  Can i put this in a follow on issue? I would put that and unit tests in follow-on jira issue

Reply:
👍